### DISCUSSION TITLE
소리 용어집

### DISCUSSION THREAD

**Discussion by designe** [euphony-io/euphony#70] [Knowledge Share]
### 소리의 질
- 풍성함;body : 재생되는 소리에서 원만함과 강건함이 느껴지는. '대담한'
- 바디 사운드;body sound : 악기의 몸통에서 나는소리. 악기가 만들어진 재료 특유의 공명으로 인해 나타나는 특징있는 소리. 비올라의 목재 질감의 소리나 금관플룻의 특징있는 소리는 목재나 백금 같은 재료들과 구별된다.
- 음조가 좋은;euphonic : 듣기에 좋은. 오디오에서 '음조가 좋다'는 말은 실제보다 과장된 풍성함을 함의한다.

### 소리를 들을때 사용하는 용어
- 가청도;audability : 음향적 불완전성의 정도를 나타내는 단위. 가청도를 작은 순서부터 나열해보면 다음과 같다. 들리지 않는, 미묘한, 근소한, 어느 정도의 분명한, 돋보이는, 그리고 으아아악!!
- 들리지 않는;inaudible : 의식적으로 지각하기에는 너무 미묘하거나 실제로는 존재하지 않는 음향적 불완전성. '의식되지 않는'과 비교될 수 있음.
- 의식되지 않는;subliminal : 너무 희미하거나 미묘해서 의식적으로 지각할 수 없는.
- 음질; quality : 재생되는 음이 완벽이라는 목표에 얼마나 가까이 있는지를 평가하는 정도
- 위상이 어긋난;phasey : 재생되는 소리가 소리의 강도와는 관련없이 귀에 어떤 압력의 감각을 만들어 낼때. 서로 위상이 연결된 두 스피커를 통해 음악을 들었을 때 많은 이들이 위상의 어긋남을 경험할 수 있음
- 선행 효과;precedence effect : 처음 들린 방향을 통해 소리가 어디서 발생했는지를 유추하는 귀의 경향성

### 소리의 성분
- 특성;characteristic : 음질 인식에 관여하는 재생음의 기본 특징. 주파수 응답, 세기, 음의 뻗어나감, 공간 표현, 해상도는 음향적 특성
- 요소;element : 음향 특성을 구성하는 부분 중 하나. 저역대, 중역대, 고역대는 주파수 응답에 대한 요소
- 임펄스; impulse : 아주 짧은 길이의 갑작스러운 신호 분출. 순간적인
- 임펄스 노이즈; impulse noise : 음반의 표면 상태 때문에 생기는 틱 잡음, 혹은 팝 잡음과 같은 순간적인 잡음
- 어택 트랜지언트; attack transient : 피아노 현, 트라이앵글, 드럼 헤드 등을 치는 소리가 내는 초기 에너지 파동
- 잡음; noise : 무엇이든 불필요한 배경 소리, 주로 무작위적이거나 음정이 없는 소리. 쉿 소리, 깨지는 소리, 틱, 팝, 휙하는 소리 등
- 디케이; decay : 음악이 그친 뒤 반향이 페이드아웃되는 것

### 주파수 대역과 관련된 용어
- 주파수 대역; frequency range : 세기 제한과는 관계 없는 주파수의 범위
- 주파수 응답; frequency response : 세기 제한을 포함하는 주파수의 범위. 예) 20~160hz+=3dB
- 초저주파; infrasonic : 가청 주파수 범위의 아래 대역. 비록 들리지 않더라도 15 ~ 20Hz 대역의 초저주파 대역은 큰 세기로 재생되면 느낄 수 있음
- 아음속; subsonic : 공기 중에서 소리의 속도보다 느린 것.
- 저주파; low frequency : 160hz 보다 낮은 주파수
- 저역; bass : 160hz보다 낮은 주파수 대역. 저음으로 간주된다
- 초저역; sub-bass : 비가청 저역
- 깊은 저역; deep bass : 40보다 낮은 주파수
- 하부 저역; low bass : 20~40Hz 사이의 대역
- mid bass : 40 ~ 80Hz
- upper base : 80 ~ 160Hz
- midrange/middles : 160 ~ 1300Hz
- lower middles : 160 ~ 320Hz
- upper middles : 650 ~ 1300Hz
- high-frequency range : 1300Hz 보다 높은 주파수 대역.
- treble : 1.3kHz 보다 높은 주파수 대역
- lower highs, middle highs, upper highs : 1.3 ~ 2.6kHz, 2.6 ~ 5kHz, 10 ~ 20kHz
- extreme highs : 10kHz 너머의 가청 주파수 대역
- ultrasonic : 인간의 가청주파수를 넘어사는 대역의 소리
- supersonic : 공기 중에서의 소리의 속도보다 빠른 것. 

### Reference
- 사운드 디자인 | Designing with sound | 제품과 서비스에 숨겨진 원리

**Comment by kuro11pow2** [comment]
와 감사합니다 소리 전문가시네요 👍 

> supersonic : 공기 중에서의 소리의 속도보다 ㅏㅃ른 것. 

마지막에서 두 번째 줄? 오타 있습니다!!

**Reply by designe** [reply]
ㅋㅋㅋ 오 오타 발견 감사합니다. 수정하겠습니다!

**Comment by ycs1m1yk** [comment]
처음보는 용어들이 많네요! 감사합니다

> * 음조가 좋은;euphonic : 듣기에 좋은. 오디오에서 '음조가 좋다'는 말은 실제보다 과장된 풍성함을 함의한다.

이게 제일 눈에 띄네욬ㅋㅋ

**Reply by designe** [reply]
ㅋㅋㅋㅋㅋ 유포니 ㅋㅋㅋ
@ycs1m1yk 혜성님이 이름짓기의 달인이라고 들었습니다.
유포니엔코도 혜성님 작품이라고 들었는데
지금 개발중이신 COVID-19 pass 서비스 이름에 대한 아이디어 있으시면 ㅋㅋ 알려주셔요 ㅋㅋㅋㅋㅋ

**Reply by ycs1m1yk** [reply]
네..? 달인이라니 잘못 알려진 것 같습니다ㅋㅋㅋㅋ ㅠㅠ
그래도 좋은 이름 떠오르면 제안해보겠습니다!! 👍 👍

**Comment by seohy02** [comment]
감사합니다! 지식이 더욱 풍부해지고 있습니닷! 💥💥💥

**Comment by psh2849** [comment]
주파수 대역도 되게 여러 가지로 나뉘네요 :hushed:

**Comment by aiclaudev** [comment]
우와 이런 유용한 전문용어들을... 감사합니다!! :star2:
